controller/search_spaces_blackbox_test.go

```diff
@@ -85,7 +85,7 @@ func (rest *TestSearchSpacesREST) TestSpacesSearchOK() {
 		{"with special chars", args{offset("0"), limit(10), "&:\n!#%?*"}, expects{totalCount(0)}},
 		{"with * to list all", args{offset("0"), limit(10), "*"}, expects{totalCountAtLeast(len(idents))}},
 		{"with multi page", args{offset("0"), limit(10), "TEST"}, expects{hasLinks("Next")}},
-		{"with last page", args{offset(strconv.Itoa(len(idents) - 1)), limit(10), "TEST"}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
+		{"with last page", args{offset(strconv.Itoa(len(idents) - 1)), limit(100), "TEST"}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
```

Comment:
I'm not sure, to be honest. I guess the DB hook did not catch those inserts, but I have not traced them further yet.